pex/util.py

```diff
@@ -200,3 +201,17 @@ def get(self, key, default=None):
   def store(self, key, value):
     with self._lock:
       self._data[key] = value
+
+
+@contextlib.contextmanager
+def named_temporary_file():
```

Comment:
should this plumb `*args, **kwargs` through to `NamedTemporaryFile` in the event someone wants to pass params to this helper in the future?